app/models/concerns/relationship_bulk.rb

```diff
@@ -106,11 +109,8 @@ def run_update_callbacks(ids_json, extra_options_json)
         # Send report if needed
         if extra_options['action'] == 'accept'
           begin Relationship.smooch_send_report(r.id) rescue nil end
-          Relationship.replicate_status_to_children(r.source_id, whodunnit, team_id)
```

Comment:
Why we don't need this line anymore? Is it because we call `apply_status_to_target`?

Reply:
Yes, Caio, I rely on callbacks rather than replicating status to all children each time.